README.md

```diff
@@ -113,10 +113,31 @@ understands.
 Local Development
 -----------------
 
-When running the test suite, you will need the following in `./tmp/htpasswd`:
+To run the tests:
+
+- `yarn install`
+- `yarn test`
+
+Prerequisites for running the tests locally:
+
+### An s3 Bucket
+
+The s3 bucket needs read/write/delete access. Set the following env variables:
+- `AWS_S3_BUCKET`
+- `AWS_ACCESS_KEY_ID`
+- `AWS_SECRET_ACCESS_KEY`
+
+### An htpasswd file
+
+When running the test suite, you will need the following in `./tmp/htpasswd` *and in the root of your s3 bucket*:
 
 ```
-test:$2y$05$t3NzHyb9qg.auYvVNZzAHuj3/e/rj6EjktdpoWzKzmal8FoH59MMK
+test:$2y$05$ZhGKbrjyUbSbiMUeYeRUKOXPKzs9./NIZHsycrQkUKIj1Z2VybqdK
```

Comment:
This is updated to match the one that is written to disk during the tests

Reply:
Is this generated per machine?

Reply:
Each time you generate it using `htpasswd` the resulting htpasswd file will be different. This is because it will use a different salt.

This key doesn't _need_ to be regenerated though, you can reuse this one every time you run the tests.